src/webviews/webview-side/interactive-common/variableExplorerGrid.less

```diff
@@ -69,7 +69,7 @@
     background-color: transparent;
     color: var(--vscode-editor-foreground);
     border-style: none;
-    padding: 0 2px;
+    padding: 0 8px;
```

Comment:
i think it's better not to hardcode 8px, instead hardcode top and bottom padding.
this way, if the 8px changes in the library or else where we don't need you worry about that... right now 8px is a magic number and it's not known why we use 8px.

Reply:
I think I like what I'm reading, but I don't fully understand!

> instead hardcode top and bottom padding

Top and bottom? Not sure how that relates to the horizontal padding.

Reply:
what i meant is, change the code to 
```css
padding-top: 0
padding-bottom: 0
```

this way the original padding of 8px is not changed.
from what i know 8px is setup by the original library, not our code, hence if the library charges tomorrow to 4px this code will need to change yet again.
ie im merely suggesting a way to future proof this and inherit the css values instead of hardcoding it.
again totally optional, which is why i didn't block this pr

Reply:
I like what you're thinking! I just didn't notice the 8px were already used by the component of my changes 👀 trying your suggestion now.

Reply:
Looks good to me! https://github.com/microsoft/vscode-jupyter/pull/10747/commits/5d5d5727a405cf84adf0f58a3952ebbb081f0d07

https://user-images.githubusercontent.com/417016/178082998-2abd5ed4-4284-4514-972d-6af4bfb6f2ed.mov